src/main/java/org/mskcc/cbio/oncokb/web/rest/SlackController.java

```diff
@@ -53,6 +56,19 @@ public SlackController(UserService userService, UserRepository userRepository, M
         this.userMapper = userMapper;
     }
 
+    private void updateUserWithRoleApiIfRequested(UserDTO userDTO) {
+        boolean apiAccessRequested = userDTO.getAdditionalInfo() != null && userDTO.getAdditionalInfo().getApiAccessRequest() != null && userDTO.getAdditionalInfo().getApiAccessRequest().isRequested();
+        if (userDTO.getLicenseType() != LicenseType.ACADEMIC || apiAccessRequested) {
+            Set<String> userDTOAuthorities = userDTO.getAuthorities();
+            userDTOAuthorities.add(AuthoritiesConstants.API);
+
+            if (userDTO.getLicenseType() == LicenseType.ACADEMIC) {
```

Comment:
Let use `equals()` method when comparing the license types. The `==` checks for reference equality.